applications/pelion_client/README.rst

```diff
@@ -0,0 +1,516 @@
+
++-------------------------------+-----------------------+--------------------------------------------------------------------------------+
+| Status                        | LED behavior          | Related application state                                                      |
++===============================+=======================+================================================================================+
+| Network backend disconnected  | LED1 blinking         | NET_STATE_DISABLED or NET_STATE_DISCONNECTED                                   |
++-------------------------------+-----------------------+--------------------------------------------------------------------------------+
+| Pelion connection search      | LED2 blinking         | PELION_STATE_DISABLED or PELION_STATE_INITIALIZED or PELION_STATE_UNREGISTERED |
++-------------------------------+-----------------------+--------------------------------------------------------------------------------+
+| Network backend connected     | LED1 solid on         | NET_STATE_CONNECTED                                                            |
++-------------------------------+-----------------------+--------------------------------------------------------------------------------+
+| Pelion connection established | LED2 solid on         | PELION_STATE_REGISTERED                                                        |
++-------------------------------+-----------------------+--------------------------------------------------------------------------------+
+
+Requirements
+************
+
+The application supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf9160dk_nrf9160ns, nrf5340dk_nrf5340_cpuapp, nrf52840dk_nrf52840
+
+.. include:: /includes/spm.txt
+
+Pelion Device Management requirements
+=====================================
+
+You need a developer account on Pelion's `Device Management Portal`_.
+
+Configuration
+*************
+
+The Pelion library is enabled with :option:`CONFIG_PELION_CLIENT` configuration option.
+
+For the library to work, you must enable and properly configure the mbed TLS library.
+To connect to Pelion's Device Management Portal, the device must be provisioned with valid credentials.
+For the firmware update procedure to work, you must provision the device with valid update resources.
+
+Mbed TLS
+========
+
+The Pelion library is depending on a properly configured mbed TLS library.
+To simplify the development within the |NCS|, a predefined set of mbed TLS configuration options was prepared.
+Set compatible with :ref:`nrfxlib:nrf_security` can be enabled by using :option:`CONFIG_PELION_NRF_SECURITY` configuration option.
+
+Pelion credentials
+==================
+
+To be able to connect to the Pelion cloud, you need to complete the following configuration:
+
+1. Generate and download a developer certificate file from Pelion's Device Management Portal.
```

Comment:
Link out to Portal when mentioning it.

Reply:
Actually, hang on, I'll find the reference for this...

Reply:
https://developer.pelion.com/docs/device-management/current/provisioning-process/provisioning-development-devices.html

General tip for linking to Pelion: Google will gladly take users to old versions, so if you can specify /current/ in links, that'll take them to the most recent version. Don't trust users to find Pelion references by Googling. It's a good way for them to get lost.